### DISCUSSION TITLE
Android port

### DISCUSSION THREAD

**Discussion by Anibalete75** [CodenameCrew/CodenameEngine#221] [Ideas]
Please do an android port

**Comment by moxie-coder** [comment]
I doubt there will be a port though, and stop reacting to your own posts, it's desperate looking and it doesn't make it any better

**Reply by rayanims** [reply]
maybe so but that just means they really want it lmao

**Comment by NexIsDumb** [comment]
I mean we won't do it officially but there are already ways to make mobile ports (both android and ios)

**Comment by ghost** [comment]
One word: N O

**Comment by r6915ee** [comment]
> Please do an android port

As far as I'm aware, Haxe supports mobile. If you were to make an Android port for Codename, though, you may need to change some things like adding touchscreen support.

**Comment by r6915ee** [comment]
Okay, it's been a while but now there's a pull request for a mobile version of Codename that's been in the works for a while now. You may want to check it out.

**Comment by callanadams42-create** [comment]
<img width="1920" height="1200" alt="XRecorderLite_24062026_223621" src="https://github.com/user-attachments/assets/34168e98-3eb8-45e9-9eb3-f01615003468" />

**Comment by callanadams42-create** [comment]
Codename Engine mods

**Comment by callanadams42-create** [comment]
https://gamebanana.com/mods/679616

**Comment by MRK2009** [comment]
Huh?

**Reply by r6915ee** [reply]
I honestly don't know, either.
